airflow/models/dag.py

```diff
@@ -2763,16 +2762,20 @@ def add_logger_if_needed(ti: TaskInstance):
                 if ti.state == TaskInstanceState.SCHEDULED:
                     if ti.next_method:
                         try:
-                            execute_callable = getattr(tasks[ti.task_id], ti.next_method)
-                            execute_callable(context={}, event=ti.next_kwargs)
+                            if isinstance(tasks[ti.task_id], MappedOperator):
+                                execute_callable = getattr(tasks[ti.task_id].operator_class, ti.next_method)
```

Comment:
You can’t do this. I’ll push some changes since it’s faster than explaining how a non-mapped operator can be obtained…